### PR TITLE
fix(editor): Fix success status for plus stalk on new canvas (no-changelog)

### PR DESCRIPTION
**Pull request by alexgrozav**
## Summary

image

## Related Linear tickets, Github issues, and Community forum posts

https://linear.app/n8n/issue/N8N-7801/all-switch-outputs-are-green

## Review / Merge checklist

- [x] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [x] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

File: packages/editor-ui/src/components/canvas/elements/handles/render-types/CanvasHandleNonMainInput.vue [hunk 1]
```diff
@@ -55,7 +53,7 @@ function onClickAdd() {
 				v-if="isHandlePlusAvailable"
 				v-show="isHandlePlusVisible"
 				:handle-classes="handleClasses"
-				:type="plusType"
+				type="ai"
```
Comment (mutdmour) [inline packages/editor-ui/src/components/canvas/elements/handles/render-types/CanvasHandleNonMainInput.vue:56]: why ai?

Comment (alexgrozav) [reply packages/editor-ui/src/components/canvas/elements/handles/render-types/CanvasHandleNonMainInput.vue:56]: For lack of a better name I guess. Got any suggestion?

Comment (mutdmour) [reply packages/editor-ui/src/components/canvas/elements/handles/render-types/CanvasHandleNonMainInput.vue:56]: what does ai stand for here? not clear to me

Comment (mutdmour) [reply packages/editor-ui/src/components/canvas/elements/handles/render-types/CanvasHandleNonMainInput.vue:56]: yes

Comment (mutdmour) [reply packages/editor-ui/src/components/canvas/elements/handles/render-types/CanvasHandleNonMainInput.vue:56]: ah okay ai for the ai nodes, sorry I thought some UI thing

Comment (alexgrozav) [reply packages/editor-ui/src/components/canvas/elements/handles/render-types/CanvasHandleNonMainInput.vue]: Renamed to `secondary`. Should be clearer now. Thanks for the suggestion!

**Review by mutdmour — DISMISSED** [review]

**Review by mutdmour — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.63.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.63.0)

**Comment by janober** [comment]
Got released with [`n8n@1.63.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.63.0)

**Pull request merged**